gen/definitions/generic/device_list.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name: Devices
+rest_endpoint: /device
+get_rest_endpoint: /device
+has_version: true
+doc_category: Monitoring - Device Details
+res_description: NOT USED - please reference the data source for device list
+attributes:
+  - model_name: deviceId
+    type: String
+    tf_name: device_id

```

Comment:
We should add example values for all attributes to have a meaningful example config in the documentation, but also have useable acceptance test.